tests/hardhat/XVS/XVSVaultFix.ts

```diff
@@ -0,0 +1,218 @@
+import { impersonateAccount, reset } from "@nomicfoundation/hardhat-network-helpers";
+import { expect } from "chai";
+import { Signer } from "ethers";
+import { ethers } from "hardhat";
+
+import { XVSVaultProxy__factory, XVSVault__factory } from "../../../typechain";
+
+const hre = require("hardhat");
+const FORK_MAINNET = process.env.FORK_MAINNET === "true";
+let FORK_ENDPOINT;
+
+const poolId = 0;
+const vaultProxy = "0x051100480289e704d20e9DB4804837068f3f9204";
+const affectedUserAddress = "0xddbc1841be23b2ab55501deb4d6bc39e3f8aa2d7";
+const affectedUserAddress2 = "0x3c7ea3ae7c47bd817c63c47e9ecee89452471a9e";
+const affectedUserAddress3 = "0x37d768c8fc5a0f54754a6bdb8b8469e6ff8cad07";
```

Comment:
I would a comment for each address, explaining their current setup in the forked contract, to understand why they are relevant and what results we should expect in the tests